[DOC] OAUTH authentication minor doc edit (#3105)
strimzi/strimzi-kafka-operator: documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc

    fallbackUserNameClaim: client_id <2>
    fallbackUserNamePrefix: client-account- <3>
    validTokenType: bearer <4>

>     userInfoEndpointUri: __https://OAUTH-SEVER-ADDRESS/auth/realms/external/protocol/openid-connect/userinfo__ <5>

Reviewer: ```suggestion
    userInfoEndpointUri: __https://OAUTH-SERVER-ADDRESS/auth/realms/external/protocol/openid-connect/userinfo__ <5>
```

Author: Typo!

----
<1> If your authorization server does not provide an `iss` claim, it is not possible to perform an issuer check. In this situation, set `checkIssuer` to `false` and do not specify a `validIssuerUri`. Default is `true`.

> <2> An authorization server may not provide a single attribute to identify both regular users and clients. A client authenticating in its own name might provide a _client ID_. But a user authenticating using a username and password, to obtain a refresh token or an access token, might provide a _username_ attribute in addition to a client ID. Use this fallback option to specify the username claim (attribute) to use if a primary user ID attribute is not available.

Reviewer: I realise this is a copy paste from upstream docs, and reading it again I think that it should be improved.

```suggestion
<2> An authorization server may not provide a single attribute to identify both regular users and clients. When a client authenticates in its own name the server might provide a _client ID_. When a user authenticates using a username and password, to obtain a refresh token or an access token, the server might provide a _username_ attribute in addition to a client ID. Use this fallback option to specify the username claim (attribute) to use if a primary user ID attribute is not available.
```

<3> In situations where `fallbackUserNameClaim` is applicable, it may also be necessary to prevent name collisions between the values of the username claim, and those of the fallback username claim. Consider a situation where a client called `producer` exists, but also a regular user called `producer` exists. In order to differentiate between the two, you can use this property to add a prefix to the user ID of the client.

> <4> (Only applicable when using `introspectionEndpointUri`) Depending on the authorization server you are using, the introspection endpoint may or may not return the _toke type_ attribute, or it may contain different values. You can specify a valid token type value that the response from the introspection endpoint has to contain.

Reviewer: I'm not sure if this was a typo - `token_type` is the literal name of the attribute returned buy the server. It seems as if you wanted to remove the underscore, but you mistakenly removed the letter 'n'. Was that your intent?
 
```suggestion
<4> (Only applicable when using `introspectionEndpointUri`) Depending on the authorization server you are using, the introspection endpoint may or may not return the _token type_ attribute, or it may contain different values. You can specify a valid token type value that the response from the introspection endpoint has to contain.
```

Author: Typo!
